pwiz_tools/Skyline/Test/ProteinAssociationTest.data/TwoProteins.fasta

```diff
@@ -0,0 +1,8 @@
+>Protein1
```

Comment:
Do we want a pattern of many unit tests having their own .data directory with small test data files in them? I think you could just write this out to a temp file directly from the code to keep the repo tidier. I love the new ability to keep things in .data dir instead of a .zip file, but think we should still use it judiciously. Like .sky files. Those would be a pain to write directly from test code (as xml I mean; the settings can be generated programmatically). As would any big DSV file.

Reply:
The only way I know to create and clean up a temporary directory is by using "TestFilesDir".
Do you know of an easier way to do that which does not require either a .zip file or .data folder?

Reply:
How about:
`using var testDir = new TemporaryDirectory(Path.Combine(TestContext.TestRunDirectory, TestContext.TestName));`
If it works we could definitely have a shortcut for that in AbstractUnitTest. :)

Reply:
Yep, that works. Thanks!
I am probably going to add a method to ProteinAssociation which takes a ProteinSource so that it can be used without any file on disk.

Reply:
Unfortunately, `TestContext.TestRunDirectory` is null when running from TestRunner.
I added a method `ProteinAssociate.UseProteinSource` so I do not need a file on disk.